it/config/src/main/content/jcr_root/apps/system/config/org.apache.felix.http.cfg.json

```diff
@@ -0,0 +1,6 @@
+{
+  "org.apache.felix.jetty.gziphandler.enable":true,
+  "org.apache.felix.jetty.gzip.includedMimeTypes":[
```

Comment:
You need to only whitelist a particular URL and not URL, please fix this.

Reply:
whitelisted the collateral path in config